models/oneinch/ethereum/unoswap/oneinch_unoswap_v5_ethereum_trades.sql

```diff
@@ -39,20 +40,20 @@ WITH unoswap AS
             - on v1 engine, the tx's were forced to amount_usd = 0 via update statement, as full refresh is less common there
         ********************************************************************************************************************/
         AND call_tx_hash not in (
-            '0x4f98ac5d5778203a0df3848c85494a179eae35befa64bb6fc360f03851385191'
-            , '0xce87a97efbf1c6c0491a72997d5239029ced77c9ef7413db66cc30b4da63fe86'
-            , '0x62c833c1ab66d17c42aeb1407755c00894f9af8691da2c2ca0f14392e3a6334c'
-            , '0x774ad4c15a6f776e71641fe4e9af3abd5bb80f7511c77548d130c2ee124ba80a'
-            , '0xad7d5814544440bdcb22760f8f2f0594958e9e6417249d96d92bf78cd05a80f5'
-            , '0xafba4b3db26b0e9f26d0ca4c709e80ee2b8bc18e3298fa67126697fc45fba0c6'
+            0x4f98ac5d5778203a0df3848c85494a179eae35befa64bb6fc360f03851385191
+            , 0xce87a97efbf1c6c0491a72997d5239029ced77c9ef7413db66cc30b4da63fe86
+            , 0x62c833c1ab66d17c42aeb1407755c00894f9af8691da2c2ca0f14392e3a6334c
+            , 0x774ad4c15a6f776e71641fe4e9af3abd5bb80f7511c77548d130c2ee124ba80a
+            , 0xad7d5814544440bdcb22760f8f2f0594958e9e6417249d96d92bf78cd05a80f5
+            , 0xafba4b3db26b0e9f26d0ca4c709e80ee2b8bc18e3298fa67126697fc45fba0c6
```

Comment:
while we're here migrating, we may as well search for new tx's which fit this criteria. you can likely query the CI table output on the dune app, order by `amount_usd` desc and see and tx's which have `amount_usd` with values blown way out of proportion (i.e. billions)

Reply:
oke will do this ser! and update!